sklearn/model_selection/_search.py

```diff
@@ -1177,6 +1177,9 @@ class GridSearchCV(BaseSearchCV):
 
         .. versionadded:: 0.20
 
+    multimetric_ : bool
+        Whether or not the scorers passed compute several metrics.
+
```

Comment:
Should
```python
         .. versionadded:: 0.24
```
be added?

Reply:
Nop it was already there. Would need to blame to know when it was introduced.